### ISSUE TITLE
Editable Inline Rows in Grid

### ISSUE THREAD

**Issue by MrCalico** [gustavnavar/Grid.Blazor#196]
Is there a way to enable inline editing for the rows of the Grid?

**Comment by gustavnavar** [comment]
No, editing is only supported using CRUD forms

**Comment by comichael** [comment]
Inline editing would be a great enhancement. Are there any plans to implement this?

Do you have any suggestions how it should be customized? I guess it's not too far from the plus/minus buttons used to modify Freight here => [https://gridblazor.azurewebsites.net/editrows](https://gridblazor.azurewebsites.net/editrows)

**Comment by SaedEzzElarab** [comment]
yes Inline editing would be a great enhancement. and i think u could make updates in the GridRowComponant.razor while make grid enable the InlineEdit replace the labels controls by Inputs controls instead
